modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/CallFunctionTest.java

```diff
@@ -147,6 +151,37 @@ public void testSetDDMFormFieldOptionsRepeatableFields() {
 				"field0", "2", "options"));
 	}
 
+	@Test
+	public void testSetDDMFormFieldValuesWithNumberOutput() {
+		DDMForm ddmForm = DDMFormTestUtil.createDDMForm("fieldName0");
+
+		DDMFormValues ddmFormValues = DDMFormValuesTestUtil.createDDMFormValues(
+			ddmForm);
+
+		ddmFormValues.addDDMFormFieldValue(
+			DDMFormValuesTestUtil.createDDMFormFieldValue(
+				"1", "fieldName0", new UnlocalizedValue("10")));
+
+		MockDDMExpressionObserver mockDDMExpressionObserver =
+			mockDDMExpressionObserver(ddmFormValues);
+
+		DDMDataProviderResponse.Builder builder =
+			DDMDataProviderResponse.Builder.newBuilder();
+
+		builder.withOutput("numberOutputId", 10);
+
+		_callFunction.setDDMFormFieldValues(
+			builder.build(),
```

Comment:
@natocesarrego we could inline this, by doing:
```
_callFunction.setDDMFormFieldValues(
    builder.withOutput(
        "numberOutputId", 10
    ).build(),
```
But since the CI has already passed, I don't think it's worth it.

Reply:
I agree with your both suggestions, @carolmariaabb! :smile: 

Thank you very much for your review. :wink: